### ISSUE TITLE
Remove/speed up zooming-animation

### ISSUE THREAD

**Issue by Glitchy-Tozier** [thetwom/Tuner#10]
Regarding usability improvements: this is absolutely the largest one I can think of.

For a few instruments this might be fine. The guitar or a violin can produce a long, loud sound. No problems there.
However, when tuning other instruments (such as the piano) the higher strings are short and not able to produce a long, constant sound. This results in the tuner in stopping to register the current note, making it jump down all the way to some low note that's humming in the background. When you hit the note again, you have to wait for the tuner to **move back** all the way to the current note.
This happens a lot (sometimes multiple times a string) and it takes up a lot of time.

#### Suggested fix: 
I won't suggest leaving out the animation completely. That might take away an important visual indicator.
Therefore, I suggest speeding up the note-switching animation, especially for cross-spectrum switches (from something like a 40Hz note to a 5000Hz note).

**Comment by Glitchy-Tozier** [comment]
Screenshot for illustration of the problem :)

![Screenshot_20210628-191315_Tuner.jpg](https://user-images.githubusercontent.com/59611881/123861190-2dc88500-d927-11eb-94be-7b8048e0fe9b.jpg)

**Comment by Glitchy-Tozier** [comment]
Another possible fix might be to "lock" the view on the current tone. Maybe by clicking on the screen, a lock would appear somewhere and the view would stick to the locked note, no matter what else is played.

**Comment by thetwom** [comment]
yes, I agree that the current behavior is not always optimal. Unfortunately this is not quickly fixed, so I have to postpone that until I find some spare time. Thanks for the suggestions anyway.

**Comment by Glitchy-Tozier** [comment]
No worries, take your time!

**Comment by thetwom** [comment]
please have a look at version [v.1.3.0](https://github.com/thetwom/Tuner/releases/tag/v1.3.0). Hopefully it solves your issue somehow:
- The plots now support touch control, meaning that you can lock/zoom/move your view according to your needs.
- Adapted animation patterns such that it zooms to the target much quicker
- Noise level detection, such that does not jump to weird values if the background noise is much higher than the tone. However, I don't know if this helps on your case since a very low noise background tone will still be detected.

**Comment by Glitchy-Tozier** [comment]
The speedup definitely is noticeable, it looks great!
